apps/events/views.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth.mixins import LoginRequiredMixin, PermissionRequiredMixin
+from django.shortcuts import get_object_or_404, redirect, render
+from django.urls import reverse_lazy
+from django.views.generic import CreateView, DeleteView, UpdateView
+
+from apps.events.forms import EventForm
+from apps.events.models import Event
+from cosmos.settings import LOGIN_URL
+
+
+def events_list(request):
+    if not request.user.is_authenticated:
```

Comment:
please reorder if statement and remove not for clarity. negation here makes no sense just creates confusion